Stripe/STPPaymentCardTextField.m

```diff
@@ -250,6 +316,60 @@ - (void)setCvcPlaceholder:(NSString * __nullable)cvcPlaceholder {
     self.cvcField.placeholder = _cvcPlaceholder;
 }
 
+- (void)setPostalCodePlaceholder:(NSString *)postalCodePlaceholder {
+    _postalCodePlaceholder = postalCodePlaceholder.copy;
+    [self updatePostalFieldPlaceholder];
+}
+
+- (void)setPostalCodeEntryEnabled:(BOOL)postalCodeEntryEnabled {
+    _postalCodeEntryEnabled = postalCodeEntryEnabled;
+    if (postalCodeEntryEnabled
+        && !self.countryCode) {
+        self.countryCode = [[NSLocale autoupdatingCurrentLocale] objectForKey:NSLocaleCountryCode];
```

Comment:
not sure where we draw the line here but maybe this can be a fallback value for the `self.viewModel.postalCodeCountryCode` getter so we don't need to remember to set it:

```objc
//  STPPaymentCardTextFieldViewModel.m

- (NSString *)postalCodeCountryCode {
  if (_postalCodeCountryCode.length) == 0 {
    _postalCodeCountryCode = [[NSLocale currentLocale] objectForKey:NSLocaleCountryCode];
  }

  return _postalCodeCountryCode;
}
```